src/tilemaker/Dockerfile

```diff
@@ -0,0 +1,94 @@
+# about what's already installed.
+
+RUN groupadd --gid 1000 node \
+  && useradd --uid 1000 --gid node --shell /bin/bash --create-home node
+
+# gpg keys listed at https://github.com/nodejs/node#release-team
+RUN set -ex \
+  && for key in \
+    9554F04D7259F04124DE6B476D5A82AC7E37093B \
+    94AE36675C464D64BAFA68DD7434390BDBE9B9C5 \
+    FD3A5288F042B6850C66B31F09FE44734EB7990E \
+    71DCFD284A79C3B38668286BC97EC7A07EDE3FC1 \
+    DD8F2338BAE7501E3DD5AC78C273792F7D83545D \
+    B9AE9905FFD7803F25714661B63B535A4C206CA9 \
+    C4F0DFFF4E8C1A8236409D08E73BC641CC11F4C8 \
+    56730D5401028683275BD23C23EFEFE93C4CFFFE \
+  ; do \
+    gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$key"; \
+  done
+
+ENV NPM_CONFIG_LOGLEVEL info
+ENV NODE_VERSION 6.10.1
+
+RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-x64.tar.xz" \
+  && curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/SHASUMS256.txt.asc" \
+  && gpg --batch --decrypt --output SHASUMS256.txt SHASUMS256.txt.asc \
+  && grep " node-v$NODE_VERSION-linux-x64.tar.xz\$" SHASUMS256.txt | sha256sum -c - \
+  && tar -xJf "node-v$NODE_VERSION-linux-x64.tar.xz" -C /usr/local --strip-components=1 \
+  && rm "node-v$NODE_VERSION-linux-x64.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt \
+  && ln -s /usr/local/bin/node /usr/local/bin/nodejs
+
+ENV YARN_VERSION 0.21.3
+
+RUN set -ex \
+  && for key in \
+    6A010C5166006599AA17F08146C2130DFD2497F5 \
+  ; do \
+    gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$key"; \
+  done \
+  && curl -fSL -o yarn.js "https://yarnpkg.com/downloads/$YARN_VERSION/yarn-legacy-$YARN_VERSION.js" \
+  && curl -fSL -o yarn.js.asc "https://yarnpkg.com/downloads/$YARN_VERSION/yarn-legacy-$YARN_VERSION.js.asc" \
+  && gpg --batch --verify yarn.js.asc yarn.js \
+  && rm yarn.js.asc \
+  && mv yarn.js /usr/local/bin/yarn \
+&& chmod +x /usr/local/bin/yarn
+
+##### end of the steps copied from Node
+
+# Install AWS CLI
+RUN set -xe && \
+    apt-get update && apt-get install -y unzip python-dev && \
+    curl -SLO "https://s3.amazonaws.com/aws-cli/awscli-bundle.zip" && \
+    unzip awscli-bundle.zip && \
+    ./awscli-bundle/install -i /usr/local/aws -b /usr/local/bin/aws && \
+    rm -r ./awscli-bundle*
+
+RUN apt-get install -y gdal-bin time
+
+# Install tl and other utilities
+RUN npm install -g \
+    tl@0.9.0 \
+    tilelive@5.12.3 \
+    mapnik@3.5.14 \
+    tilelive-mapnik@0.6.18 \
+    tilelive-file@0.0.3 \
+    tilelive-s3@6.5.0
+
+##### Include the django container so we can send status updates
+# Copy django requirements (will copy the rest later, so that any change to
+# the code doesn't trigger re-running the whole provisioning step)
+COPY ./django/requirements.txt /opt/pfb/django/requirements.txt
+
+RUN apt-get install -y \
+    python-dev \
+    python-pip
+
+# Copy django requirements (will copy the rest later, so that any change to
```

Comment:
Whoops, yup, made a mess of that.  Was copying pieces from the analysis dockerfile but rearranging them to make the "include Django container" part self-contained.